src/makeRobotAccountant.js

```diff
@@ -7,6 +7,20 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+  const errorCheck = (first) => {
```

Comment:
try to be more descriptive with the names 
```suggestion
  const errorCheck = (firstNumber) => {
```